### PR TITLE
Make the course overview/motivation less aggressive

### PR DESCRIPTION
**Pull request by tekknolagi**
There is room for pointing out flaws without putting down extant
curricula.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -33,16 +33,13 @@ to program: it teaches them to break down problems and solve them in terms of
...
-source control, build systems, and testing infrastructure-that makes efficient
-development and maintenance of large software systems possible.
-
-Even tools that students are exposed to, like the POSIX shell, are not
-presented as things to be learned and understood. Instead, our courses treat
-them as obstacles to be brushed aside as quickly as possible so students can
-get back to the "real work" of programming. We give students pre-written build
-scripts and rote `g++` commands to memorize and don't expect them to know or
-care what's going on behind the scenes.
+curriculum does not spend much time focusing on the ubiquitous tooling -- like
```
Comment (tchebb) [inline index.md:36]: Bring back the Unicode em dashes please :)

File: index.md [hunk 2]
```diff
@@ -33,16 +33,13 @@ to program: it teaches them to break down problems and solve them in terms of
...
+source control, build systems, and testing infrastructure -- that makes
+efficient development and maintenance of large software systems possible.
+
+Even tools that students are exposed to, like the POSIX shell, are not part of
+the explicit learning objectives for the existing courses. Instead, our
+courses treat them as obstacles in the "real work" of programming.
```
Comment (tchebb) [inline index.md:42]: "obstacles to"?

**Comment by tchebb** [comment]
LGTM otherwise.

File: index.md (hunk 1 above)
Comment (tekknolagi) [reply index.md:36]: ASCII LYFE

Comment (tekknolagi) [reply index.md:36]: In all seriousness, MD should render to em-dash

Comment (tchebb) [reply index.md:36]: I didn't think that was part of standard Markdown, but if it works it works. Can we still get rid of the spaces though?

File: index.md (hunk 2 above)
Comment (tekknolagi) [reply index.md:42]: Using as example in https://www.merriam-webster.com/dictionary/obstacle

**Pull request merged**